### PR TITLE
Check function declarations in Caffe2 code

### PR DESCRIPTION
**Pull request by cyyever**
Fixes #ISSUE_NUMBER

### CODE REVIEW

File: caffe2/perfkernels/embedding_lookup_idx_avx2.cc [hunk 1]
```diff
@@ -5,9 +5,10 @@
 //// DO NOT MODIFY!!!
```
Comment (ezyang) [inline caffe2/perfkernels/embedding_lookup_idx_avx2.cc:5]: See comment here?

Comment (cyyever) [reply caffe2/perfkernels/embedding_lookup_idx_avx2.cc:5]: While the file was generated from python, it has been modified several times, as you can see from the git history..

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**